src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,19 @@
 
   @include hidden;
 
-  bottom: 2em;
+  align-items: flex-end;
+  display: flex;
   font-size: 1.2em;
-  left: 3em;
+  height: 86%;
+  justify-content: center;
+  margin: auto 20%;
   pointer-events: none;
   position: absolute;
-  right: 3em;
   text-align: center;
-  transition: bottom $animation-duration-short ease-out;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: height $animation-duration-short ease-out;
+  width: 60%;
```

Comment:
This won't work well when the video aspect ratio is e.g. 1:2 (portrait) and the width rather small, there's not much space left for the subtitles and 40% of the width are wasted. Please don't assume that all players are in 16:9 format.